doc/additionalDoc/package.h

```diff
@@ -101,7 +101,7 @@ but in general the external data is based upon the sensor values provided by a "
 For this reason the signal evaluations are realized through the cascade of dependencies and start from the evaluation of an input
 signal of a periodic node (in general the device). This is realized inside a \b real-time thread.
 
-To add flexibility to a node, it is possible to add command with arguments to modify the internal behavior of the entity 
+To add flexibility to a node, it is possible to add command with arguments to modify the internal behavior of the entity
```

Comment:
Those changes pollutes the difference for nothing. Would you mind setting up your editor to avoid such changes next time ? Thanks.

Reply:
You are right, those changes should not appear in this commit. If you want to review without those, you still can use the `-w` option of `git diff`, or in the Github UI, add `?w=1` in the url:
https://github.com/stack-of-tasks/dynamic-graph/pull/27/files?w=1

I could setup my editor to avoid those changes, or (better), I should use [the script](https://github.com/nim65s/dotfiles/blob/master/.config/fish/config.fish#L90) I already made for this case.

But I would really prefer not having to do this. Trailing whitespaces should just not have been commited in the first place. And people should setup their editors (or git hooks) to avoid pushing that.

I would prefer to make commits that just get rid of them instead,

Reply:
> I would prefer to make commits that just get rid of them instead

I am not strongly against removing trailing white spaces, although it pollutes the history. I think they should at least be in a separate commit. But don't change this commit, it is merely a remark.